configs/JHEF722HD/config.h

```diff
@@ -0,0 +1,121 @@
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F7X2
+
+#define BOARD_NAME        JHEF722HD
+#define MANUFACTURER_ID   JHEF
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_ICM42688P   
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACCGYRO_BMI270
+#define USE_BARO_DPS310
+#define USE_BARO_BMP280
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+#define USE_MAX7456
+
+#define BEEPER_PIN           PC13
+#define MOTOR1_PIN           PC8
+#define MOTOR2_PIN           PC9
+#define MOTOR3_PIN           PA8
+#define MOTOR4_PIN           PA9
+#define MOTOR5_PIN           PB0
+#define MOTOR6_PIN           PB1
+#define MOTOR7_PIN           PA10
+#define MOTOR8_PIN           PB4
+#define LED_STRIP_PIN        PB3
+#define UART1_TX_PIN         PB6
+#define UART2_TX_PIN         PA2
+#define UART4_TX_PIN         PA0
+#define UART5_TX_PIN         PC12
+#define UART6_TX_PIN         PC6
+#define UART1_RX_PIN         PB7
+#define UART2_RX_PIN         PA3
+#define UART4_RX_PIN         PB11
+#define UART5_RX_PIN         PD2
+#define UART6_RX_PIN         PC7
+#define I2C1_SCL_PIN         PB8
+#define I2C1_SDA_PIN         PB9
+#define LED0_PIN             PC14
+#define LED1_PIN             PC15
+#define SPI1_SCK_PIN         PA5
+#define SPI2_SCK_PIN         PB13
+#define SPI3_SCK_PIN         PC10
+#define SPI1_SDI_PIN         PA6
+#define SPI2_SDI_PIN         PB14
+#define SPI3_SDI_PIN         PC11
+#define SPI1_SDO_PIN         PA7
+#define SPI2_SDO_PIN         PB15
+#define SPI3_SDO_PIN         PB5
+#define ADC_VBAT_PIN         PC1
+#define ADC_CURR_PIN         PC3
+#define PINIO1_PIN           PC0
+#define PINIO2_PIN           PC2
+#define FLASH_CS_PIN         PA15
+#define MAX7456_SPI_CS_PIN   PB12
+#define GYRO_1_EXTI_PIN      PC4
+#define GYRO_2_EXTI_PIN      PB10
+#define GYRO_1_CS_PIN        PA4
+#define GYRO_2_CS_PIN        PB2
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP( 0, PC8,  2,  0) \
+    TIMER_PIN_MAP( 1, PC9,  2,  0) \
+    TIMER_PIN_MAP( 2, PA8,  1,  0) \
+    TIMER_PIN_MAP( 3, PA9,  1,  0) \
+    TIMER_PIN_MAP( 4, PB0,  2,  0) \
+    TIMER_PIN_MAP( 5, PB1,  2,  0) \
+    TIMER_PIN_MAP( 6, PA10, 1,  0) \
+    TIMER_PIN_MAP( 7, PB4,  1,  0) \
+    TIMER_PIN_MAP( 8, PB3,  1,  0) \
+
+#define ADC3_DMA_OPT 0
+
+#define MAG_I2C_INSTANCE  (I2CDEV_1)
+#define BARO_I2C_INSTANCE (I2CDEV_1)
+
+#define FLASH_SPI_INSTANCE   SPI3
+#define GYRO_1_SPI_INSTANCE  SPI1
+#define GYRO_2_SPI_INSTANCE  SPI1
```

Comment:
SPI sharing is not recommended and can lead to dead locks.

Reply:
Thank you for your suggestion. We will avoid this problem in the next board design, but now our products have been launched to the market, and there are no major problems in current use, so I think we can apply for merger.